packages/sanity/src/_internal/cli/commands/manifest/manifestGroup.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+  name: 'manifest',
+  signature: '[COMMAND]',
+  isGroupRoot: true,
+  description: 'Interacts with the studio configuration.',
```

Comment:
Is this description correct?

Reply:
Yes, but I was a bit unsure about what to put here. Manifest sort of looks at the whole studio config and extracts part of it.

Reply:
For context `schemaGroup` reads: `Interacts with Sanity Studio schema configurations`